cpp/src/io/parquet/reader_impl_chunking_utils.cuh

```diff
@@ -188,17 +188,25 @@ std::vector<row_range> compute_page_splits_by_row(device_span<cumulative_page_in
  *
  * @param chunks List of column chunk descriptors
  * @param pass_pages List of page information for the pass
+ * @param pass_page_mask Boolean page mask indicating which pass pages to decompress. Empty
```

Comment:
Please skip changes in this file. These are only added from PR #19132 so that this PR is also in current and working state. These will vanish when the other PR merges